src/main/resources/messages/AiCommitsBundle.properties

```diff
@@ -109,3 +109,7 @@ settings.huggingface.token.example=hf_fKASPPYLkasgjasKwpSnAASRdasdCdAsddsASSDF
 settings.huggingface.maxNewTokens=Max new tokens
 settings.huggingface.waitForModel=Wait for model
 settings.huggingface.waitModel.comment=When a model is warm, it is ready to be used, and you will get a response relatively quickly. However, some models are cold and need to be loaded before they can be used. In that case, you will get a 503 error. Rather than doing many requests until it is loaded, you can wait for the model to be loaded.
+
+
+settings.openRouter.token.example=sk-or-v1-...
+settings.openRouter.token.comment=Your OpenRouter API key. You can find it in your OpenRouter account settings.
```

Comment:
Could you maybe add a link (with <a href...>) to the account settings? Check how it's done for the others.